### DISCUSSION TITLE
My company wont let me use codewhisperer, but I was using Fig

### DISCUSSION THREAD

**Discussion by fordneild** [aws/codewhisperer-command-line-discussions#102] [General]
Just something to note: after switching to cw, i got a dm from the infosec guys at my company saying cw wasn't allowed. It seems to me that cw is trying to do to much, and as a result, im forced to throw the baby out with the bath water. I just redownloaded original fig, and am happy again. Please dont get rid of it entirely! 

P.S. congrats on everything, these are good problems to have

**Comment by grant0417** [comment]
Is there something specific that the infosec person had an issue with? The product should not operate too differently from Fig so if there is any specific issues lets us know.

**Comment by brendanfalk** [comment]
Hey Ford good to hear from you! We're really happy to help here. Do you want to connect me to the infosec team at Anduril and I will help you get it approved.

If Anduril is willing to approve Fig, I can guarantee they will approve an AWS product
